documentation/get-started/tutorials/my-first-project.en.md

```diff
@@ -0,0 +1,91 @@
+---
+title: My first project
+long_title: My first project using QGIS and QField with QFieldCloud
+tx_slug: documentation_get-started_tutorials_myfirstproject
+---
+
+# My first project
+
+This page will go through a step by step example of how to prepare a simple QGIS project containing a GeoPackage and a basemap.
+It will include the configuration of a simple attribute form and the styling of the feature layer.
+
+## Step-by-step Example
+
+**Used case**
+
+[Maya Mielena](../../assets/images/maya_waving.gif)
+
+*Hi, my name is Maja Mielena, I am a retired GIS Specialist and I will show you how to prepare a simple QGIS project so that you can use it inside the QField application*
+*We will create a Geopackage containing a point layer and a line layer.*
+*This will allow you to go out into nature and record your personal points of interests and track yourself using the line layer.*
+
+*Let's get started!*
+
+### QGIS Project - Adding new data
+
+1. Open a new QGIS Project on your desktop.
+2. Add a basemap of your choice to the project.
+
+    1. Under the browser on the top left of the QGIS window you will find the dropdown *XYZ Tiles*.
+    2. From there you can add the OpenStreetMap basemap, which is the most commonly used basemap.
+3. We will now locate our area of interest.
+There exists a very useful QGIS plugin called *OSM Place Search*, which allows you to browse through the different place features available in the OpenStreetMap data.
+    1. Under *Plugins* in the Menu toolbar direct to *Manage and Install Plugins*
+    2. In the search bar type *OSM Place Search* and install the plugin
+    3. A new window should appear in the side panel where you can enter your place of interest
+    4. Add it to your project as a feature by clicking on the middle button at the bottom of the plugin interface.
+    5. A new feature layer will be available as a temporary layer in your project.
+4. Adding a new layer
+
+        Great, now that we have located our area of interest, we can add a new feature layer to the project.
+
+    1. Click on *Layer* and  create a new *Vector Layer*
+    2. Give your Gpkg a general name.
+    Additionally give a name to the table (which we are about to create).
+
+        **To Note**: A Gpkg can consist of several layers, including points, lines, polygons and raster layers.
```

Comment:
There's a native way to create notes:

https://github.com/opengisch/QField-docs/blob/1e09ff8b932ffab2baf0a9846e6e59efcd5e04e9/documentation/how-to/plugins.en.md?plain=1#L18-L20